ui/components/Header.js

```diff
@@ -354,18 +388,19 @@ function K8sContextMenu({
                 }
                 {contexts?.contexts?.map(ctx => {
                   const meshStatus = getMeshSyncStatus(ctx.id);
+                  const brokerStatus = getBrokerStatus(ctx.id);
                   const operStatus = getOperatorStatus(ctx.id);
 
                   function getStatus(status) {
                     if (status) {
-                      return "Active"
+                      return status
```

Comment:
```suggestion
                      return STATUS.ACTIVE
```

Reply:
> @MUzairS15 From the ss - The MeshSync Status is only showing the port number, not the status. It should show Active since it's connected.

The IP address indicates, it's active, otherwise status will be something else (Disabled)

Reply:
Okay, Gotcha. 
The above suggestion is valid, right?